### DISCUSSION TITLE
Is there a way of getting summary information for WS socket binary data?

### DISCUSSION THREAD

**Discussion by chrisguest75** [mitmproxy/mitmproxy#6700] [General]
I'm using the mitm proxy web portal and it gets really clunky if you're sending lots of binary data over a WS.  I'm not so interested in the actual bits and bytes just a summary of the time, size of data and direction. 

I'm usually running the docker container.  Is it possible?

**Comment by Prinzhorn** [comment]
You could implement a custom [contentview](https://github.com/mitmproxy/mitmproxy/blob/main/examples/addons/contentview.py)

**Reply by chrisguest75** [reply]
Thanks.  I'll have a look at it.
